script/django/setup.bat

```diff
@@ -0,0 +1,7 @@
+pushd django
+python3 -m venv .
+CALL .\Scripts\activate.bat
+.\Scripts\python.exe -m pip install -r requirements.txt
```

Comment:
Not sure if it was an issue with my machine but I needed to call the local virtual environment `.exe` otherwise it couldn't find the dependencies of the app